src/server/test/models/PushSubscriptionModel-test.ts

```diff
@@ -21,92 +23,71 @@ const PUSH_SUBSCRIPTION_2 = {
 };
 const CONTENT_ENCODINGS = ['aes128gcm', 'aesgcm'];
 
-test.beforeEach(() => {
-  const subscriptions =
-      pushSubscriptionModel.getSubscriptionsForUser(USER_LOGIN);
-  if (subscriptions) {
-    subscriptions.forEach((subscriptionInfo) => {
-      pushSubscriptionModel.removePushSubscription(
-          USER_LOGIN, subscriptionInfo.subscription);
-    });
+test.beforeEach(async () => {
+  // See https://cloud.google.com/docs/authentication/production
+  if (!process.env.GOOGLE_APPLICATION_CREDENTIALS) {
```

Comment:
Hmm, maybe we should just do this in dash-server or something. Fine for now though.